### PR TITLE
add button style

### PR DESCRIPTION
**Pull request by ArnelaKapllani**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,31 +1,79 @@
...
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <header>
       <h1>Quiz App</h1>
     </header>
-    <div class="flex-container1">
-      <p>What properties flip the axes in flexbox? 
+    <section>
+      <div class="flex-container1">
```
Comment (NilsOt1) [inline index.html]: Maybe you could find an element, that fits better than div. Maybe another section or article. You could use this for every div you used in your code :)

File: index.html [hunk 2]
```diff
@@ -1,31 +1,79 @@
...
+        <p>What are the colors of the sunflower?</p>
+        <button class="sunflower-answer" type="button">
+          Here is the answer
+        </button>
+        <div class="list-horizontal">
+          <ul id="list">
```
Comment (NilsOt1) [inline index.html]: Try to use classes with BEM instead of ID's. You will have less problems with the cascade

File: index.html [hunk 3]
```diff
@@ -1,31 +1,79 @@
...
+      <div class="flex-container2">
+        <p>what are the colors of the rainbow?</p>
+        <button class="rainbow-answer" type="button">This is the answer</button>
+        <img
+          src="https://images.app.goo.gl/1wTfjtwYRCx1nREQ6"
+          class="full-bookmark-icon"
+        />
+      </div>
+    </section>
+    <footer>
```
Comment (NilsOt1) [inline index.html:78]: The footer should be one of the last elements in your html code

File: index.html [hunk 4]
```diff
@@ -1,31 +1,79 @@
...
-
-
+      <div class="list-horizontal">
+        <ul id="list">
+          <li><a href="html">#html</a></li>
+          <li><a href="flexbox">#flexbox</a></li>
+          <li><a href="css">#css</a></li>
+        </ul>
+      </div>
+      cd
```
Comment (NilsOt1) [inline index.html]: Do you use this? If not, you could delete it

File: style.css [hunk 5]
```diff
@@ -14,4 +14,45 @@ h1 {
 .flex-container1 {
   display: flex;
   background-color: lightsalmon;
+  flex-direction: column;
+}
+
+.sunflower-answer {
+  background-color: chocolate;
+  border: 0cqmax;
```
Comment (NilsOt1) [inline style.css]: I think this value ist not valid. You could define the style, the width and the color of the border here

File: style.css [hunk 6]
```diff
@@ -14,4 +14,45 @@ h1 {
...
+  text-decoration: none;
+  display: flex;
+  font-size: 12px;
+  margin: auto;
+  cursor: pointer;
+  height: 0px;
+  width: 100px;
+  justify-content: center;
+  border-radius: 10px;
+  border: none;
```
Comment (NilsOt1) [inline style.css]: watch the last comment

File: style.css [hunk 7]
```diff
@@ -14,4 +14,45 @@ h1 {
...
+  box-shadow: 0px 0px 2px 2px rgb(151, 108, 10);
+}
+
+p {
+  text-align: center;
+  color: rgba(60, 12, 12, 0.811);
+  font-size: large;
+}
+
+ul#list {
```
Comment (NilsOt1) [inline style.css]: You could use a class for this list instead of an ID

File: style.css [hunk 8]
```diff
@@ -14,4 +14,45 @@ h1 {
...
+  list-style: none;
+  padding-top: 20px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.list-horizontal {
+  display: flex;
+  margin: 0auto;
+  border: 0cqmax;
```
Comment (NilsOt1) [inline style.css]: See line 22

**Review by NilsOt1 — COMMENTED** [review]
Some minor suggestions :)

File: index.html (hunk 1 above)
Comment (ArnelaKapllani) [reply index.html]: Thanks for the review and suggestions @NilsOt1

**Pull request merged**